tests/test_tacacs_server_group.rb

```diff
@@ -231,11 +241,14 @@ def test_remove_server_twice_tacacs
     assert_empty(servers, 'Error: Collection not empty')
 
     detach_aaaservergroup(aaa_group)
-    detach_tacacsserverhost(server1)
-    detach_tacacsserverhost(server2)
+    detach_tacacsserverhost(server_name1)
+    detach_tacacsserverhost(server_name2)
   end
 
   def test_get_vrf_tacacs
+    # Not part of NetDev development
```

Comment:
@mikewiebe I'll add the excludes to the YAML file and return based on validate_property_excluded. Does this sound ok? Not sure if the negative testing makes sense for all non-netdev method as some properties have multiple methods. Thanks!